### PR TITLE
feat: display disabled actions on action sheet

### PR DESCRIPTION
**Pull request by reinaldonetof**
## Proposed changes

> We can still show the items as disabled, but also add a tiny (short hopefully) text that tells them why.

#### Before
Screen Shot 2023-11-21 at 15 28 01

Screen Shot 2023-11-21 at 15 28 18

#### After

Screen Shot 2023-11-22 at 15 37 21

Screen Shot 2023-11-22 at 15 36 52

## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] Improvement (non-breaking change which improves a current function)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update (if none of the other choices apply)

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/RocketChat/Rocket.Chat/blob/develop/.github/CONTRIBUTING.md#contributing-to-rocketchat) doc
- [x] I have signed the [CLA](https://cla-assistant.io/RocketChat/Rocket.Chat.ReactNative)
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if applicable)
- [ ] I have added necessary documentation (if applicable)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

[TC-974]

[TC-974]: https://rocketchat.atlassian.net/browse/TC-974?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by diegolmello — CHANGES_REQUESTED** [review]
Tests are failing

File: app/lib/constants/colors.ts [hunk 1]
```diff
@@ -110,6 +110,8 @@ export const colors = {
 		strokeExtraLight: '#EBECEF',
 		strokeLight: '#CBCED1',
 		surfaceTint: '#F7F8FA',
+		fontTitlesLabels: '#1F2329',
```
Comment (GleidsonDaniel) [inline app/lib/constants/colors.ts:113]: Where are you using this color?

File: app/i18n/locales/pt-BR.json [hunk 2]
```diff
@@ -757,5 +757,6 @@
   "The_user_wont_be_able_to_type_in_roomName": "O usuário não poderá digitar em {{roomName}}",
   "The_user_will_be_able_to_type_in_roomName": "O usuário poderá digitar em {{roomName}}",
   "Enable_writing_in_room": "Permitir escrita na sala",
-  "Disable_writing_in_room": "Desabilitar escrita na sala"
+  "Disable_writing_in_room": "Desabilitar escrita na sala",
+  "You_dont_have_permission": "Você não tem permissão para realizar esta ação. Verifique com o proprietário da sala."
```
Comment (GleidsonDaniel) [inline app/i18n/locales/pt-BR.json:761]: `You_dont_have_permission` for what?
This label is "misleading", there is no problem with placing large labels, as long as they bring coherent meanings to the context.
Maybe: `You_dont_have_permission_to_perform_this_action`

**Review by GleidsonDaniel — CHANGES_REQUESTED** [review]

File: app/lib/constants/colors.ts (hunk 1 above)
Comment (reinaldonetof) [reply app/lib/constants/colors.ts:113]: Nice catch! This color was related to previous design!

**Review by GleidsonDaniel — APPROVED** [review]

**Review by diegolmello — COMMENTED** [review]
Left a few comments. My only concern is if it's going to be too annoying, but we could merge and get feedbacks.

**Review by diegolmello — APPROVED** [review]

**Pull request merged**